kolibri/core/assets/src/api_resource.js

```diff
@@ -254,7 +411,6 @@ class Resource {
   getModel(id, data = {}) {
     let model;
     if (!this.models[id]) {
-      Object.assign(data, { [this.idKey]: id });
```

Comment:
Well, it may be that the model exists on the server, we just need to fetch it. `createModel` we should reserve for cases where the model does not exist on the server.